include/libsleigh.hh

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright (c) 2021-present, Trail of Bits, Inc.
+  All rights reserved.
+
+  This source code is licensed in accordance with the terms specified in
+  the LICENSE file found in the root directory of this source tree.
+*/
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-copy"
```

Comment:
Not sure what the equivalent warnings are for MSVC. I don't want to guess without testing on Windows since I might unnecessarily silence some warnings, so let's wait until #6 to address this.